inc/widgets/largo-series-posts.php

```diff
@@ -123,13 +126,21 @@ function form( $instance ) {
 		</p>
```

Comment:
The option here is labeled "show date" but instead shows the full byline, without date. Should this be fixed by relabeling the option or changing the output?

Reply:
That's... odd. I would think we should relabel the option to `Show byline on posts?` since that's what is on the Recent Posts widget and we're trying to get this one to resemble that one.

Reply:
@benlk I updated the label in https://github.com/INN/largo/pull/1734/commits/cba0714ffb48b2d6751418d9918cf14ff1b9f61b.